recipes/pandas-vet/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "pandas-vet" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 57ea4906c176d61ea9c186607be7a53f4f1561b53f19f5b76ce2a5050f0bad41
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - attrs
+    - flake8 >3.0.0
+    - python >=3.6
+    - setuptools
+
+test:
+  imports:
+    - pandas_vet
+  requires:
+    - coveralls >=1.5.1
+    - pytest >4.0.0
+    - pytest-cov >=2.6.1
+
+about:
+  home: https://github.com/deppen8/pandas-vet
+  license: MIT
+  license_family: MIT
```

The MIT licence requires a copy of the text to be included in all distributions of the package. Ideally it should be included in the tarball but as it's missing you can also include a copy in the recipe directory.
```suggestion
  license_family: MIT
  licence_file: LICENCE
```